syntax = "proto2";

package annotator;

message IFMessage {
  required string serverName = 2;
  oneof srcIpAddr {
    fixed32 srcV4 = 3;
    bytes srcV6 = 4;
  }
  required int32 srcPort = 5;

  oneof dstIpAddr {
    fixed32 dstV4 = 6;
    bytes dstV6 = 7;
  }
  required int32 dstPort = 8;
  required uint32 timestamp_s = 9;
  required uint32 timestamp_ms = 10;
  required uint32 ipVersion = 11;
}